docker-compose/run_selenium_tests_local_keybase.sh

```diff
@@ -0,0 +1,213 @@
+exit 1
+fi
+
+# Check temp certstore dir and create if not existing
+if [ -d "${CERTSTORE_TEMPORARY_MOUNT_PATH}" ]
+then
+echo_msg
+echo_msg "  - OK: The temporary certstore mount path is found at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+else
+mkdir ${CERTSTORE_TEMPORARY_MOUNT_PATH}
+echo_msg
+echo_msg "  - OK: Created the temporary certstore mount path at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+fi
+
+# Keybase cert files
+keybase_certfiles="${keybase_env_path}/${KEYBASE_CERTFILES_SUBPATH}"
+keybase_cert_file="${keybase_certfiles}/${CERT_FILENAME}"
+keybase_key_file="${keybase_certfiles}/${KEY_FILENAME}"
+
+# Check that certfiles in keybase dir exist
+if [ -f "${keybase_cert_file}" ]
+then
+    echo_msg
+    echo_msg "  - OK: The cert file was found at: ${keybase_cert_file}"
+else
+    echo_msg
+    echo_msg "ERROR: The cert file could NOT be found at: ${keybase_cert_file}"
+    exit 1
+fi
+
+if [ -f ${keybase_key_file} ]
+then
+    echo_msg
+    echo_msg "  - OK: The key file was found at: ${keybase_key_file}"
+else
+    echo_msg
+    echo_msg "ERROR: The key file could NOT be found at: ${keybase_key_file}"
+    exit 1
+fi
+
+# Copy certfiles from KeyBase to local for container mount
+echo_msg "  - COPY certfiles from KeyBase to local temp for container mount..."
+echo_msg
+cp ${keybase_cert_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+cp ${keybase_key_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
+
+# stop all before run selenium tests
+docker-compose -f docker-compose.selenium.yml down
+
+export DJANGO_USER_ID_SALT=${DJANGO_USER_ID_SALT}
+export DJANGO_USER_ID_ITERATIONS=${DJANGO_USER_ID_ITERATIONS}
+export DJANGO_PASSWORD_HASH_ITERATIONS=${DJANGO_PASSWORD_HASH_ITERATIONS}
+export DJANGO_SLSX_CLIENT_ID=${DJANGO_SLSX_CLIENT_ID}
+export DJANGO_SLSX_CLIENT_SECRET=${DJANGO_SLSX_CLIENT_SECRET}
+export HOSTNAME_URL=${HOSTNAME_URL}
+export USE_MSLSX=${USE_MSLSX}
+export DJANGO_MEDICARE_SLSX_REDIRECT_URI=${DJANGO_MEDICARE_SLSX_REDIRECT_URI}
+export DJANGO_MEDICARE_SLSX_LOGIN_URI=${DJANGO_MEDICARE_SLSX_LOGIN_URI}
+export DJANGO_SLSX_HEALTH_CHECK_ENDPOINT=${DJANGO_SLSX_HEALTH_CHECK_ENDPOINT}
+export DJANGO_SLSX_TOKEN_ENDPOINT=${DJANGO_SLSX_TOKEN_ENDPOINT}
+export DJANGO_SLSX_SIGNOUT_ENDPOINT=${DJANGO_SLSX_SIGNOUT_ENDPOINT}
+export DJANGO_SLSX_USERINFO_ENDPOINT=${DJANGO_SLSX_USERINFO_ENDPOINT}
+
+echo "Selenium tests ..."
+
+docker-compose -f docker-compose.selenium.yml run tests bash -c "python runtests.py --selenium ${TESTS_LIST}"
+
+# Remove certfiles from local directory
+echo_msg
+echo_msg Shred and Remove certfiles from CERTSTORE_TEMPORARY_MOUNT_PATH=${CERTSTORE_TEMPORARY_MOUNT_PATH}
+echo_msg
+
+if which shred
+then
+  echo_msg "  - Shredding files"
+  shred "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+  shred "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
+fi
+
+echo_msg "  - Removing files"
+rm "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+rm "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
```

Comment:
Might be good to also stop all docker containers once the run is done.

Reply:
on second though, might need to keep the containers to attach and trouble shoot...
e.g. check loggings, docker-compose -f docker-compose.selenium.yml logs > full_log.txt.

Reply:
take that back, added logic to stop services, it's considered rare needing to do trouble shoot